api/CSSSupportsRule.json

```diff
@@ -5,57 +5,54 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSSupportsRule",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": "41"
           },
           "chrome_android": {
-            "version_added": false
+            "version_added": "41"
           },
           "edge": {
-            "version_added": "≤18",
-            "version_removed": "79"
+            "version_added": "12"
           },
           "firefox": {
-            "version_added": "17",
-            "notes": "From Firefox 17 to 19, methods and properties were defined on <code>CSSSupportsRule</code>. From version 20, they were on <code>CSSConditionRule</code>.",
-            "flags": [
```

Comment:
Can you mention this removal in the description, as requested by @ddbeck in another PR?